### PR TITLE
Documented site_model.csv

### PR DESCRIPTION
**Pull request by micheles**

### CODE REVIEW

File: doc/manual/oqum/hazard/01d_config_classical.tex [hunk 1]
```diff
@@ -146,19 +146,32 @@
...
+can just skip the field. Also, most of the GMPEs are not sensitive to
+it.  The same can be said for the \texttt{backarc} parameter: $0$
+means false and this is the default, so you can skip such column. All
+columns that are not needed by the GMPEs your are using can be safely
+skipped, while you will get an error if a relevant column is missing.
+
 If the closest available site with soil conditions is at a distance greater than 5~km from the investigation location, a warning is generated.
 
+\textbf{Note:} For backward-compatibility reasons, the site model file can
+also be given in XML format. That old format is deprecated but there are
```
Comment (raoanirudh) [inline doc/manual/oqum/hazard/01d_config_classical.tex]: Should we still keep the xml format documented for legacy users?

File: doc/manual/oqum/hazard/01d_config_classical.tex [hunk 2]
```diff
@@ -146,19 +146,32 @@
...
 \begin{listing}[htbp]
-  \inputminted[firstline=1,firstnumber=1,fontsize=\footnotesize,frame=single,linenos,bgcolor=lightgray]{xml}{oqum/hazard/verbatim/input_site_model.xml}
+  \inputminted[firstline=1,firstnumber=1,fontsize=\footnotesize,frame=single,linenos,bgcolor=lightgray]{xml}{oqum/hazard/verbatim/input_site_model.csv}
   \caption{Example site model input file}
   \label{lst:input_site_model}
 \end{listing}
 
+Notice that the $0$ for the field \texttt{vs30measured} means that the
+\texttt{vs30} field is inferred, not measured. This is the default, so you
+can just skip the field. Also, most of the GMPEs are not sensitive to
```
Comment (raoanirudh) [inline doc/manual/oqum/hazard/01d_config_classical.tex]: Suggest removing "Also, most of the GMPEs are not sensitive to it."

File: doc/manual/oqum/hazard/01d_config_classical.tex [hunk 3]
```diff
@@ -146,19 +146,32 @@
...
+it.  The same can be said for the \texttt{backarc} parameter: $0$
+means false and this is the default, so you can skip such column. All
+columns that are not needed by the GMPEs your are using can be safely
```
Comment (raoanirudh) [inline doc/manual/oqum/hazard/01d_config_classical.tex]: *you are using

File: doc/manual/oqum/hazard/01d_config_classical.tex (hunk 1 above)
Comment (micheles) [reply doc/manual/oqum/hazard/01d_config_classical.tex]: I would not do that. Legacy users will already have their way to produce site model files in the old format and that will keep working. On the other hand, we do not want to give bad ideas to the new users.

File: doc/manual/oqum/hazard/01d_config_classical.tex (hunk 2 above)
Comment (micheles) [reply doc/manual/oqum/hazard/01d_config_classical.tex]: Done

File: doc/manual/oqum/hazard/01d_config_classical.tex (hunk 3 above)
Comment (micheles) [reply doc/manual/oqum/hazard/01d_config_classical.tex]: Fixed

**Pull request merged**